pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, git, fetchgit }:
+
+stdenv.mkDerivation {
+  name = "ble.sh";
+  version = "0.3.3";
+
+  nativeBuildInputs = [ git ];
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "refs/tags/v0.3.3";
+    hash = "sha256-Gfo2S1t5Kdy+8TEDS4M5yhyRShvzQIljdE0MQK1CL+4=";
+    fetchSubmodules = true;
+  };
+
+  doCheck = true;
+
+  postInstall = ''
+    mkdir -p "$out/bin"
+    echo 'source "$(dirname "$BASH_SOURCE")/../share/ble.sh"' >"$out/bin/ble.sh"
```

Comment:
A symbolic link is sufficient here.

Reply:
Does symlink work? I saw some other files and directories emitted under `share` directory. If ble.sh depends on them, I'm afraid symlink won't work here.

Reply:
Nope, I wanted to make it sure that it's independent beforehand. Then we can remove some output files/dirs at `share`, which is irreverent to make it work. I'll see it.

Reply:
I didn't mean ble.sh doesn't depend on the data files in `share`, but it will correctly find the data directory even if it is sourced through a symbolic link.